Automattic/wp-calypso
# Update ranges in `package.json` to match `yarn.lock`

#### Changes proposed in this Pull Request

This PR changes the ranges in `package.json` files to match whatever version is actually installed (fetched from `yarn.lock`, with the help of https://www.npmjs.com/package/syncyarnlock)

This has the following benefits:

* It is often easier (and I guess more common among devs) to check `package.json` to see the version of a dependency, even when the source of truth is `yarn.lock`. This change makes both numbers closer.

* Avoid publishing packages that accidentally use features from newer versions. Example: imagine a package depends on `lib@^1.0.0` and yarn has installed `lib@1.2.3` and the developer uses a feature introduced in `lib@1.2.0`. If we publish the package, a consumer may use `lib@1.1.0` and it will break. Keeping versions in `package.json` up to date will help to mitigate this problem.

This change will introduce a new package, because `eslint-config-prettier@^6.10.1` was already used and pointing to `eslint-config-prettier@6.11.0`

```diff
< Before
< wp-calypso-monorepo@0.17.0 eslint-config-prettier@6.10.1
< wp-calypso-monorepo@0.17.0 eslint-config-prettier@6.10.1 get-stdin@6.0.0
---
> After
> wp-calypso-monorepo@0.17.0 eslint-config-prettier@6.11.0
> wp-calypso-monorepo@0.17.0 eslint-config-prettier@6.11.0 get-stdin@6.0.0
```


#### Testing instructions

* Clone the branch, install deps and generate a package list (`find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list`)
* Do the same with master (I used commit 3547b37ebe for this test).
* Diff both outputs and verify there are no changes in the dependency tree introduced by this branch (other than the specified above).
